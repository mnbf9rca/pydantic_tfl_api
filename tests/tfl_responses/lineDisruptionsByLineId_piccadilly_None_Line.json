{"status_code": 200, "headers": {"Date": "Mon, 15 Jul 2024 15:03:06 GMT", "Content-Type": "application/json; charset=utf-8", "Transfer-Encoding": "chunked", "Connection": "keep-alive", "Cache-Control": "public, must-revalidate, max-age=30, s-maxage=60", "Via": "1.1 varnish", "Content-Encoding": "gzip", "Age": "0", "Access-Control-Allow-Headers": "Content-Type", "Access-Control-Allow-Methods": "GET,POST,PUT,DELETE,OPTIONS", "Access-Control-Allow-Origin": "*", "Api-Entity-Payload": "Disruption", "X-Backend": "api", "X-Cache": "MISS", "X-Cacheable": "Yes. Cacheable", "X-Frame-Options": "deny", "X-Proxy-Connection": "unset", "X-TTL": "60.000", "X-TTL-RULE": "0", "X-Varnish": "2026256364", "X-AspNet-Version": "4.0.30319", "X-Operation": "Line_DisruptionByPathIds", "X-API": "Line", "CF-Cache-Status": "DYNAMIC", "Strict-Transport-Security": "max-age=31536000", "Set-Cookie": "_cfuvid=Rh4KtFPTSJzvCweOLQeRuImQImn8iQhWI91bsEBSl3g-1721055786072-0.0.1.1-604800000; path=/; domain=.tfl.gov.uk; HttpOnly; Secure; SameSite=None", "Server": "cloudflare", "CF-RAY": "8a3aa3267a128883-LHR"}, "url": "https://api.tfl.gov.uk/Line/piccadilly/Disruption", "content": "[{\"$type\":\"Tfl.Api.Presentation.Entities.Disruption, Tfl.Api.Presentation.Entities\",\"category\":\"RealTime\",\"type\":\"routeBlocking\",\"categoryDescription\":\"RealTime\",\"description\":\"Piccadilly Line: No service between Rayners Lane and Uxbridge due to a points failure at Uxbridge. GOOD SERVICE on the rest of the line. To continue your journey beyond Rayners Lane, please change at Rayners Lane for the Metropolitan line. \",\"affectedRoutes\":[],\"affectedStops\":[],\"closureText\":\"partSuspended\"}]"}