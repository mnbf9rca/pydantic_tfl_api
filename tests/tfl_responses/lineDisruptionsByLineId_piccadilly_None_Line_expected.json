["{\"category\":\"RealTime\",\"disruption_type\":\"routeBlocking\",\"category_description\":\"RealTime\",\"description\":\"Piccadilly Line: No service between Rayners Lane and Uxbridge due to a points failure at Uxbridge. GOOD SERVICE on the rest of the line. To continue your journey beyond Rayners Lane, please change at Rayners Lane for the Metropolitan line. \",\"summary\":null,\"additional_info\":null,\"created\":null,\"last_update\":null,\"affected_routes\":[],\"affected_stops\":[],\"closure_text\":\"partSuspended\",\"content_expires\":\"2024-07-15T15:41:32Z\",\"shared_expires\":\"2024-07-15T15:42:02Z\"}"]